docs/data/date-pickers/date-picker/date-picker.md

```diff
@@ -25,12 +25,20 @@ Check-out their documentation page for more information:
 - [Date Field](/x/react-date-pickers/date-field/)
 - [Date Calendar](/x/react-date-pickers/date-calendar/)
 
-## Uncontrolled vs. Controlled
+## Uncontrolled vs. controlled value
 
 The component can be uncontrolled or controlled.
```

Comment:
I think that we should clarify, "The component can be uncontrolled or controlled." is confusing to me, it could be the value, the open state, the current value, or any other state of the component. How about?
```suggestion
The component has multiple states that can be controlled, see how the value can be controlled and uncontrolled in this demo:
```

Reply:
I would not talk about "multiple states" here since we are only demonstrating the value control.

I would just do:

```suggestion
The component value can be uncontrolled or controlled.
```

Reply:
Fair enough